.github/workflows/deploy_release.yml

```diff
@@ -53,24 +53,16 @@ jobs:
     - name: apt
       run: |
         sudo apt-get update \
-          && sudo apt-get install -y swig3.0
-    - name: pip
+          && sudo apt-get install -y swig
```

Comment:
Do we have Ubuntu20 meanwhile? On the old one, "swig" package was version 1 or 2, I think

Reply:
Oh didn't realise that, non of the other scripts install `swig3.0`. 

> swig is already the newest version (3.0.12-1).

seems fine, no/

Reply:
Oh, seems that got fixed... smart move... didn't check in quite a while...